docs/DSA-Problem-Solution/Loop in linked list.md

```diff
@@ -0,0 +1,265 @@
+---
+id: isomorphic-strings
+title: "What are isomorphic strings."
+sidebar_label: "Isomorphic strings"
+sidebar_position: 6
+description: "Isomorphic strings are two strings that can be transformed into each other by a consistent mapping of characters. "
+tags: [String,isomorphic strings]
+---
```

Comment:
change `id`, `title`, `sidebar_label`, `description`, and `tags` based on your file name and contents